azurerm/resource_arm_automation_datetime_variable.go

```diff
@@ -0,0 +1,42 @@
+package azurerm
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceArmAutomationDatetimeVariable() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmAutomationDatetimeVariableCreateUpdate,
+		Read:   resourceArmAutomationDatetimeVariableRead,
+		Update: resourceArmAutomationDatetimeVariableCreateUpdate,
+		Delete: resourceArmAutomationDatetimeVariableDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: AutomationVariableCommonSchemaFrom(map[string]*schema.Schema{
+			"resource_group_name": resourceGroupNameSchema(),
+
+			"value": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
You should validate this, there is an existing function for it

Reply:
Fixed